62_CAD/main.cpp

```diff
@@ -68,7 +68,7 @@ constexpr std::array<float, (uint32_t)ExampleMode::CASE_COUNT> cameraExtents =
 	600.0,	// CASE_8
 };
 
-constexpr ExampleMode mode = ExampleMode::CASE_7;
```

Comment:
revert these changes please, I changed shader compilation a lot, it will introduce conflicts